lib/grpc_reflection/service.ex

```diff
@@ -0,0 +1,18 @@
+defmodule GrpcReflection.Service do
```

Comment:
This is the point where we know we are working with Agent, and callers are insulated from that decision